presto-hive/src/test/java/io/prestosql/plugin/hive/metastore/cache/TestCachingHiveMetastore.java

```diff
@@ -57,9 +59,8 @@
     public void setUp()
     {
         mockClient = new MockThriftMetastoreClient();
-        MetastoreLocator metastoreLocator = new MockMetastoreLocator(mockClient);
+        ThriftHiveMetastore thriftHiveMetastore = createThriftHiveMetastore();
```

Comment:
it looks like changes here are not needed by the actual change, am I missing something? Either extract them as separate commit (or even PR because I am not sure if they are worth doing) or simply drop.

Reply:
I'm using the createThriftHiveMetastore method when constructing the CachingHiveMetastore, as I don't want to introduce any stubs and using actual implementation seems fine here.